data/transactions/logic/assembler.go

```diff
@@ -1072,6 +1072,16 @@ func (ops *OpStream) assemble(fin io.Reader) error {
 		err := fmt.Errorf("unknown opcode %v", opstring)
 		return lineErr(ops.sourceLine, err)
 	}
+
+	// backward compatibility: do not allow jumps behind last instruction in TEAL v1
+	if ops.Version <= AssemblerDefaultVersion {
```

Comment:
Shouldn't this simply be `<= 1`

Reply:
`AssemblerDefaultVersion` will change, but we want to pin this at <=1

Reply:
done